content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,168 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Policy reports provide information about policy execution and violations. Use '--policy_report' with the apply command to generate policy report.
```

Comment:
Anything that users are expected to type with respect to the CLI or anything else follows a convention of mono-spaced type. In Markdown, this is in code formatting, so all those flags like --policy_report and -r etc need to be `--policy_report` and `-r` etc.

Reply:
`apply`